app/src/androidTest/java/com/github/fribourgsdp/radio/MainActivityTest.kt

```diff
@@ -74,7 +74,7 @@ class MainActivityTest {
     fun correctTransitionToFireBaseTestActivity(){
```

Comment:
The name of the test is no longer in line with what the test does.

Reply:
This was solved when merging from master.